src/Nethermind/Nethermind.Runner/Properties/launchSettings.json

```
@@ -70,6 +70,13 @@
         "ASPNETCORE_ENVIRONMENT": "Development"
       }
     },
+    "Mekong": {
+      "commandName": "Project",
+      "commandLineArgs": "-c mekong --data-dir .data --jsonrpc-enabled true",
```

Got overwritten somehow. Again:

```suggestion
      "commandLineArgs": "-c mekong --data-dir .data",
```